src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
1. i see no point in "is_reusable" parameter. UCP could behave as if is_reusable==1 all the time.
1. is it possible to achieve the same purpose with "struct" type instead of IOV (which has a handle)?